src/models/post.rs

```diff
@@ -9,6 +9,12 @@ use url::Url;
 // Validation
 const MAX_SHORT_CONTENT_LENGTH: usize = 2000;
 const MAX_LONG_CONTENT_LENGTH: usize = 50000;
+// Reserved keyword used by the system to mark deleted posts with relationships
+const RESERVED_CONTENT_DELETED: &str = "[DELETED]";
+const MAX_ATTACHMENTS: usize = 3;
+const MAX_ATTACHMENT_URL_LENGTH: usize = 200;
+// Allowed protocols for attachment URLs: pubky://, http://, https://, ipfs://, ipns://
+const ALLOWED_ATTACHMENT_PROTOCOLS: &[&str] = &["pubky", "http", "https", "ipfs", "ipns"];
```

Comment:
If we explicitly define a list, it signals we currently have support for these protocols. Do we?

If not, might be better to restrict to the protocols we do support.